.github/workflows/benchmark_asterinas.yml

```diff
@@ -11,7 +11,11 @@ jobs:
     runs-on: self-hosted
     strategy:
       matrix:
-        benchmark: [sysbench-cpu, sysbench-thread, getpid]
+        benchmark: |
+          [
+          sysbench-cpu, sysbench-thread, getpid,
+          lmbench-getpid, lmbench-fork, lmbench-signal
```

Comment:
What about `lmbench-read` and `lmbench-write`? And are they any other bench programs in lmbench that are missing in this list? If any, why?

Reply:
I see. Then I suggest adding the two tests to the list, but comment them out with a TODO that details the reason why they are disabled for now.